src/js_native_api_v8.cc

```diff
@@ -621,27 +594,50 @@ inline bool CanBeHeldWeakly(v8::Local<v8::Value> value) {
 
 }  // end of anonymous namespace
 
+void Finalizer::ResetEnv() {
```

Comment:
Would it help to mark small functions like this as `inline`?

Reply:
It should not make any difference here. Modern compilers mostly ignore the `inline` keyword. They make decision based on the function size when the code is visible to compiler. I.e. these functions will be inlined anyway.